CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ and apply your changes there.
 
 - Ensure the PR description clearly describes the problem and solution. If there is an existing GitHub issue open describing this bug, please include it in the description so we can close it.
 
-- Ensure the PR is based on the `dev` branch of the Omniperf GitHub repository.
+- Ensure the PR is based on the `amd-staging` branch of the Omniperf GitHub repository.
 
 - Omniperf requires new commits to include a "Signed-off-by" token in the commit message (typically enabled via the `git commit -s` option), indicating your agreement to the projects's [Developer's Certificate of Origin](https://developercertificate.org/) and compatability with the project [LICENSE](https://github.com/ROCm/omniperf/blob/main/LICENSE):
```

Comment:
Will the URL on this link (i.e. LICENSE file) need to chnage too?

Reply:
I could change this to a relative link. For example,

```suggestion
- Omniperf requires new commits to include a "Signed-off-by" token in the commit message (typically enabled via the `git commit -s` option), indicating your agreement to the projects's [Developer's Certificate of Origin](https://developercertificate.org/) and compatability with the project [LICENSE](LICENSE):
```

Reply:
Awsome! That's even better.